### DISCUSSION TITLE
参考文献无法显示

### DISCUSSION THREAD

**Discussion by duruoping** [sjtug/SJTUThesis#1032] [提问]
![image](https://github.com/sjtug/SJTUThesis/assets/113845026/4446f9e5-dea3-41af-baf9-a85c26945722)
报错信息如图

**Comment by LogCreative** [comment]
橘黄色的只是一个 warning，可能你的编译工具不是 latexmk (xelatex)；或者你的文档的其他部分出现了红色 error 级别的错误，导致没有正确地进行多次编译。

**Reply by duruoping** [reply]
感谢感谢，我重新调整了setting.json之后就可以了
